README.md

```
@@ -1,193 +1,290 @@
+                          (StateFlow<UIState>)
+                                  │
+                                  ▼
+                        Jetpack Compose UI (screens)
+```
+
+---
+
+## Tech Stack
+
+### UI
+| Library | Version | Purpose |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)